src/ppom-controller.ts

```diff
@@ -429,7 +432,7 @@ export class PPOMController extends BaseControllerV2<
   #subscribeMessageEvents(): void {
     const onNetworkChange = this.#onNetworkChange.bind(this);
     this.messagingSystem.subscribe(
-      'NetworkController:stateChange',
```

Comment:
Is this necessary to upgrade `network-controller`? If not, thoughts on doing this in another PR?

Reply:
@mcmire changes removed and created this issue ticket https://github.com/MetaMask/ppom-validator/issues/191